### PR TITLE
React-icons will be supported

### PR DESCRIPTION
**Pull request by ashwinKumar0505**

### CODE REVIEW

File: src/utils/stringToIconConvertor.ts [hunk 1]
```diff
@@ -1,13 +1,36 @@
+// @ts-nocheck
 import React from 'react'
+import * as fontAwesomeIcons from 'react-icons/fa'
```
Comment (sanandnarayan) [inline src/utils/stringToIconConvertor.ts:3]: by doing this you are importing all the icons in react - icons.

Comment (ashwinKumar0505) [reply src/utils/stringToIconConvertor.ts:3]: As we are dynamically importing the icons and also user can choose any icon from the font awesome. So we can not limit the icons. That's why i am using this way of importing bro

Comment (sanandnarayan) [reply src/utils/stringToIconConvertor.ts:3]: https://medium.com/@magbicaleman/intro-to-dynamic-import-in-create-react-app-6305bb397c46
https://reactjs.org/docs/code-splitting.html
https://nextjs.org/docs/advanced-features/dynamic-import
https://www.npmjs.com/package/react-dynamic-import

**Comment by sanandnarayan** [comment]
approved

**Pull request merged**